src/animated-binding.js

```diff
@@ -116,9 +116,11 @@ Binding.extend(AnimatedBinding, {
   animateOut: function(node, callback) {
     if (node.firstViewNode) node = node.firstViewNode;
 
-    this.animateNode('out', node, function() {
-      if (callback) callback.call(this);
-    });
+    this.animateNode('out', node, this.animateOutHelper.bind(this, callback));
+  },
+
+  animateOutHelper: function(callback) {
+    if (callback) callback.call(this);
```

Comment:
Because `animateNode` has `if (callback) callback.call(this);` on line 161 below, it seems this code is pointless. You can remove `animateOutHelper` and just add the callback on line 119.

`this.animateNode('out', node, callback);`